wpilibjExamples/networktables.json.bck

```diff
@@ -0,0 +1,18 @@
+[
```

Comment:
This file should be deleted (or at least removed from source control via `git rm --cached`).